src/main/java/org/kohsuke/stapler/idea/JellyLanguageInjector.java

```diff
@@ -44,11 +44,15 @@ public void getLanguagesToInject(@NotNull final MultiHostRegistrar registrar, @N
             Language language = findLanguage("CSS");
             if (language == null) return;
 
-            registrar.startInjecting(language);
-            registrar.addPlace("dummy_selector {","}",
-                    (PsiLanguageInjectionHost)value,
-                    TextRange.from(1, value.getTextLength() - 2));
-            registrar.doneInjecting();
+            try {
+                registrar.startInjecting(language);
+                registrar.addPlace("dummy_selector {", "}",
+                                   (PsiLanguageInjectionHost) value,
+                                   TextRange.from(1, value.getTextLength() - 2));
+                registrar.doneInjecting();
+            } catch (IllegalArgumentException ignored) {
+                // occurs when syntax is malformed, like style==""
+            }
```

Comment:
Why rely on exceptions to handle this? It is, after all, an exception about this plugin not preparing the parameters correctly for an API. We can probably just add another `if` checking that `textLength` is greater than 2, i.e. there are two quotes and something in them.

Reply:
Right @duemir , pushed another commit!